docs/src/language_servers.md

```diff
@@ -627,6 +627,23 @@ Follow installation instructions on [LSP-metals](https://github.com/scalameta/me
     }
     ```
 
+### Solidity
```

Comment:
This is under the "Shell" programming language section. Use 2 hashes if it's a separate programming language.

Reply:
Oh, my bad, will fix 👍